views/groceries.ejs

```diff
@@ -0,0 +1,29 @@
+<%- include("partials/head.ejs") %>
+<%- include("partials/header.ejs") %>
+<h2>Groceries List</h2>
+    <table id="groceries-table">
+      <tr id="groceries-table-header">
+        <th>Name</th>
+        <th>Status</th>
+        <th colspan="2"></th>
+      </tr>
+      <% if (groceries && groceries.length) { %>
+        <% groceries.forEach((product) => { %>
+            <a href="/groceries/update/<%= product.id %>"><button type="button">edit</button></a>
```

Comment:
This line should be inside the <td colspan="2"> element. Move it after line 16.
ex:
```
<td colspan="2">
                <a href="/groceries/update/<%= product.id %>"><button type="button">edit</button></a>
            <form method="POST" action="/groceries/delete/<%= product.id %>" style="display: inline">
                <button>delete</button>
                <input type="hidden" name="_csrf" value="<%= _csrf %>" />
            </form>
        </td>
```